tests/test_middleware.py

```diff
@@ -18,6 +19,13 @@
     SlotPolicy,
     SplashCookiesMiddleware,
     SplashDeduplicateArgsMiddleware,
+    SplashHtmlResponse,
+    SplashTextResponse,
+)
+
+splash_scrapy_content_types = (
+    (SplashTextResponse, TextResponse, b'text/*'),
+    (SplashHtmlResponse, HtmlResponse, b'text/html'),
```

Comment:
:lipstick: Since these lines are not too long, and this variable is only used once, maybe the variable could be removed and its content moved below, inline?